Client/Frontend/Home/Pocket/FxHomePocketViewModel.swift

```diff
@@ -105,17 +105,43 @@ class FxHomePocketViewModel {
 
     // MARK: - Private
 
+    func updatePocketStoryViewModels(with stories: [PocketStory]) {
+        self.pocketStoriesViewModels = []
+        for story in stories {
+            self.bind(pocketStoryViewModel: .init(story: story))
+        }
+    }
+
     private func getPocketSites(completion: @escaping () -> Void) {
         pocketAPI
             .globalFeed(items: UX.numberOfItemsInSection)
             .uponQueue(.main) { [weak self] (pocketStory: [PocketFeedStory]) -> Void in
```

Comment:
Could it be possible to avoid using deferred here? Simply using completions instead. With new code we try to avoid using it.